static/redfish/v1/schema/OemLogEntry_v1.xml

```diff
@@ -0,0 +1,40 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
I also think this commit is needed. (https://github.com/ibm-openbmc/bmcweb/pull/472)
```
https://github.com/ibm-openbmc/bmcweb/commit/2101f4e97ffd125ba9e8a49eda3458ff0920c7a3
```

Reply:
will add

Reply:
done